docs/DEPLOYMENT.md

```diff
@@ -428,6 +429,19 @@ curl -X POST -H "Authorization: <access-token>" \
 ```
 Replace `<access-token>` with your JWT token. you must have `config-refresh` scope to call this API. 
 
+## Deploying on Disconnected Cluster
+
+- To deploy on a disconnected cluster you need to fork the [tektoncd/catalog][catalog] or you can use your own catalog but it must follow the [Catalog TEP][catalog-tep].
+- Fork the tektoncd/hub and update the [hub config][hub-config] file for your catalog details.
```

Comment:
Could you please add/explain more or add an example of block of what needs to be updated in the config file ?